src/content/docs/en/guides/integrations-guide/prefetch.mdx

```diff
@@ -1,161 +1,11 @@
-import prefetch from '@astrojs/prefetch';
-
-export default defineConfig({
-  // ...
-  integrations: [
-    prefetch({
-      // Only prefetch links with an href that begins with `/products`
-      selector: "a[href^='/products']",
-    }),
-  ],
-});
-```
-
-### config.intentSelector
-
-By default, the prefetch script also searches the page for any links that include a `rel="prefetch-intent"` attribute, ex: `<a rel="prefetch-intent" />`. This behavior can be changed in your `astro.config.*` file to use a custom query selector when finding prefetch-intent links.
-
-```js
-// astro.config.mjs
-import { defineConfig } from 'astro/config';
-import prefetch from '@astrojs/prefetch';
-
-export default defineConfig({
-  // ...
-  integrations: [
-    prefetch({
-      // Only prefetch links with an href that begins with `/products` or `/coupons`
-      intentSelector: ["a[href^='/products']", "a[href^='/coupons']"],
-
-      // Use a string to prefetch a single selector
-      // intentSelector: "a[href^='/products']"
-    }),
-  ],
-});
-```
-
-### config.throttle
-
-By default the prefetch script will only prefetch one link at a time. This behavior can be changed in your `astro.config.*` file to increase the limit for concurrent downloads.
-
-```js
-// astro.config.mjs
-import { defineConfig } from 'astro/config';
-import prefetch from '@astrojs/prefetch';
-
-export default defineConfig({
-  // ...
-  integrations: [
-    prefetch({
-      // Allow up to three links to be prefetched concurrently
-      throttle: 3,
-    }),
-  ],
-});
-```
-
-## Troubleshooting
-
-*   If your installation doesn't seem to be working, try restarting the dev server.
-*   If a link doesn't seem to be prefetching, make sure that the link is pointing to a page on the same domain and matches the integration's `selector` option.
-
-For help, check out the `#support` channel on [Discord](https://astro.build/chat). Our friendly Support Squad members are here to help!
-
-You can also check our [Astro Integration Documentation][astro-integration] for more on integrations.
-
-## Contributing
-
-This package is maintained by Astro's Core team. You're welcome to submit an issue or PR!
-
-## Changelog
-
-See [CHANGELOG.md](https://github.com/withastro/astro/tree/main/packages/integrations/prefetch/CHANGELOG.md) for a history of changes to this integration.
+:::note[removed]
+`@astrojs/prefetch` has been replaced by the [built-in `prefetch` feature](/en/guides/prefetch/) introduced in Astro 3.5. See the [migration guide](/en/guides/prefetch/#migrating-from-astrojsprefetch) for instructions on updating an older project.
 
-[astro-integration]: /en/guides/integrations-guide/
+If you still are using this integration in pre-v3.5 Astro project, you can find [its previous documentation](https://github.com/withastro/astro/blob/c47478bbf6b21973419f25234c68efb59466b368/packages%2Fintegrations%2Fprefetch%2FREADME.md) from an older version of Astro.
```

Comment:
“its previous documentation” struck me as not great for a link label, so I think updating that to “the `@astrojs/prefetch` README” would be a nice a11y win. Also suggesting some tweaks because I found “previous docs”/“older version” a bit brain bending, but it is late, so my brain may be bend-prone.

```suggestion
If you are still using this integration in a pre-v3.5 Astro project, you can read an archived copy of [the `@astrojs/prefetch` README](https://github.com/withastro/astro/blob/c47478bbf6b21973419f25234c68efb59466b368/packages%2Fintegrations%2Fprefetch%2FREADME.md) on GitHub.
```